src/main/java/com/example/capstone_project/controller/responses/term/get_termDetail/TermDetailResponse.java

```diff
@@ -1,7 +1,6 @@
-package com.example.capstone_project.controller.responses.term.get;
+package com.example.capstone_project.controller.responses.term.get_termDetail;
```

Comment:
Change package name from get_termDetail into getTermDetail, get_plans to getPlans for correct Java naming convention